data/remnant/remnant missions.txt

```diff
@@ -4495,6 +4495,7 @@ mission "Remnant: Expanded Horizons Astral job"
 	source
 		government "Remnant"
 	waypoint
+		attributes "ember waste"
```

Comment:
```suggestion
```

The Astral jobs are specifically supposed to draw on the entire accessible galaxy.

Reply:
ahh okok

Reply:
also i commited it on the wrong branch